_data-prepper/monitoring.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Monitoring
+nav_order: 33
+---
+
+# Monitoring Data Prepper with metrics
+
+Data Prepper can be monitored with metrics using [Micrometer](https://micrometer.io/). There are two types of metrics: JVM and system metrics and plugin metrics. [Prometheus](https://prometheus.io/) is used as the default metrics backend.
+
+## JVM and system metrics
+
+JVM and system metrics are runtime metrics used to monitor Data Prepper instances. They include metrics for classloaders, memory, garbage collection, threads, and others. For more information, see [JVM and system metrics](https://micrometer.io/docs/ref/jvm). 
+
+### Naming
+
+JVM and system metrics follow predefined names in [Micrometer](https://micrometer.io/docs/concepts#_naming_meters), such as `jvm_classes_loaded` and `jvm_memory_used`. 
```

Comment:
I didn't find `jvm_classes_loaded` or `jvm_memory_used` on that page. Maybe it's just that they follow naming conventions?
```suggestion
JVM and system metrics follow [Micrometer naming conventions](https://micrometer.io/docs/concepts#_naming_meters). Example names are `jvm_classes_loaded` and `jvm_memory_used`. 
```

Reply:
@dlvenable Please clarify.

Reply:
That page gives an example of `http.server.requests`. But, we don't report that for system metrics. So we use `jvm.memory.used` as an example.

Here is my attempt to clarify:

> VM and system metrics follow predefined names in [Micrometer](https://micrometer.io/docs/concepts#_naming_meters). For example, the Micrometer metrics names for memory usage is `jvm.memory.used`. Micrometer will change the name to match the metric system. Following the same example, `jvm.memory.used` is reported to Prometheus as `jvm_memory_used` and reported to CloudWatch as `jvm.memory.used.value`.